### ISSUE TITLE
Element not selected then navigating with keyboard through <sui-select> results

### ISSUE THREAD

**Issue by uNki23** [edcarroll/ng2-semantic-ui#33]
Hey everybody!

First of all: great project!!

I discovered a behaviour that's different from original Semantic UI. 

If you navigate through the results of `<sui-select>` with keyboard up and down arrow keys, the current element is not selected. Hence the result list is not scrolling and you can not see where you are in the result list if you select anything out of the visible result range.

The original Semantic UI behaviour is different. If you navigate through the results with up and down key, the element is selected and the list is scrolling if you reach the upper or lower end of the visible range. 

[http://semantic-ui.com/modules/dropdown.html#search-selection](http://semantic-ui.com/modules/dropdown.html#search-selection) --> select dropdown and navigate with keyboard keys

[http://edcarroll.github.io/ng2-semantic-ui/#/components/select](http://edcarroll.github.io/ng2-semantic-ui/#/components/select) --> won't work here

You can argue about whether or not it's a good idea to directly **select** the item when navigating with keyboard. But it's absolutely essential to scroll the list.

Thank you!

**Comment by edcarroll** [comment]
Hi there, apologies for the delay getting back to you on this one. I've been thinking about how to implement this within the new dropdown & select implementations, and can be done from within the dropdown menu. I'll have a look at getting this sorted in the next few weeks, but can't promise anything I'm afraid.

**Comment by edcarroll** [comment]
I've just merged a change that contains a fix for a part of this issue (`0.7.0`) - specifically, as you rightly say, that the keyboard navigation should auto-scroll the select whenever the highlighted item is out of the viewport.

Now, the behaviour is that it is scrolled automatically when navigating with the keyboard. However, I have opted to not add the auto-selection when navigating - let me know if that is something you are really keen to see in the project.
